launch/launch/__init__.py

```diff
@@ -75,8 +78,11 @@ def __init__(self, cmd, name, output_handler, exit_handler, env=None):
 
     def send_signal(self, signal):
         if self.transport:
+            self.task_state.signals_received.append(signal)
             self.transport.send_signal(signal)
 
     def terminate(self):
-        if self.transport:
+        if os.name != 'nt':
+            self.send_signal(signal.SIGINT)
```

Comment:
OK, that's fine. I can change this pull request so that the `terminate` method remains as is and add a new function that tries to call `SIGINT` first, e.g. `request_exit`, so that the names match the behavior.

Because of the problem described in the description of this PR, I need a way to call SIGINT after the match condition is met, because gcov doesn't produce output for SIGTERM'd process.

So what do you think should happen to processes that receive SIGINT after the output was matched? Is the basic idea in this PR acceptable? Or do you have an alternative suggestion?